src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figuresArray = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figuresArray.length; i++) {
+            if (i > Math.floor(figuresArray.length / 2)) {
```

Comment:
Instead of using `Math.floor(figuresArray.length / 2)` directly in the condition, consider defining it as a constant at the beginning of the method or class for better readability and to avoid magic numbers.